subiquity/server/controllers/filesystem.py

```diff
@@ -1456,21 +1456,28 @@ def start(self):
         self._start_task = schedule_task(self._start())
 
     async def _start(self):
-        context = pyudev.Context()
-        self._monitor = pyudev.Monitor.from_netlink(context)
-        self._monitor.filter_by(subsystem="block")
-        self._monitor.enable_receiving()
-        self.start_listening_udev()
+        self.pyudev_context = pyudev.Context()
```

Comment:
Could you declare this in the initializer as an `Optional[pyudev.Context]` ?

Reply:
Done